src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+header {
```

Comment:
do not style tags, styles classes instead

Reply:
where do i need to add padding?